lib/auth/auth_with_roles.go

```diff
@@ -1112,6 +1112,19 @@ func (a *AuthWithRoles) ChangePasswordWithToken(ctx context.Context, req ChangeP
 	return a.authServer.ChangePasswordWithToken(ctx, req)
 }
 
+// CreateUser inserts a new entry in a backend.
```

Comment:
`// CreateUser inserts a new __user__ entry in a backend.`
Similar for other method comments and in `.proto`

Reply:
revised